src/pages/LoginToAdminPage.js

```diff
@@ -36,40 +36,57 @@ function LoginToAdminPage() {
         setPlayersList(tempArray);
     }
     
+    const getSuccessfullFechedData = async (password) => {
+        setPassword(password)
+        const result = await fetch(
+            `${apiUrl}/admin/players/`,
+            {   
+                headers: { 'Authorization': `Token ${password}`}
+            });
+        let responseData = await result.json();
+        if (result.status === 200) {
+            responseData = responseData.map(item => {
+                return {
+                    ...item,
+                    selected: false
+                };
+            })
+            setPlayersList(responseData);
+            setAuthorized(true);
+        };
+        if (result.status === 403) {
+            setShowPasswordErrorMessage(true)
+        };
+    }
+
     const getPlayersDataHandler = async (password) => {
         const adminPassword = password
         console.log('adminPassword', adminPassword)
-        setPassword(adminPassword)
 
         setIscallLoading(true);
         setShowPasswordErrorMessage(false);
 
         try {
-            const result = await fetch(
-                `${apiUrl}/admin/players/`,
-                {   
-                    headers: { 'Authorization': `Token ${adminPassword}`}
-                });
-            let responseData = await result.json();
-            if (result.status === 200) {
-                responseData = responseData.map(item => {
-                    return {
-                        ...item,
-                        selected: false
-                    };
-                })
-                setPlayersList(responseData);
-                setAuthorized(true);
-            };
-            if (result.status === 403) {
-                setShowPasswordErrorMessage(true)
-            };
+            getSuccessfullFechedData(adminPassword)
```

Comment:
`getSuccessfullFechedData` is `async` function. When the function is `async`, we need to call it with `await`. 
If you don't use `await` here, the program doesn't wait for `getSuccessfullFechedData` to be done but just go to the next line. (It will still call the `getSuccessfullFechedData` so it would work but we will have unexpected bugs)

and there is `setIscallLoading(false);` below, which will stop the loading indicator UI before it's actually done. And also `try/catch` can't catch the error from the async function because it would already pass the block.